missions:

  mission_1:
    description: "Template"
    # filepaths are relative to the `autonomy_engine/trajectory_dir` parameter
    filepaths:
      - "template_dir/template_trajectory.csv"
    # for a full list of implemented entities see 'include/autonomy_core/autonomy_defs.h'
    entities_actions:
      - ["px4_gps", "continue"]
      - ["px4_imu", "failure"]
      - ["px4_mag", "continue"]
      - ["px4_bar", "continue"]
      - ["mission_cam", "hold"]
      - ["uwb", "hold"]
      - ["lrf", "hold"]
      - ["optitrack", "land"]
